docs/source/python/integration/python_r.rst

```diff
@@ -0,0 +1,315 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+.. or more contributor license agreements.  See the NOTICE file
+.. distributed with this work for additional information
+.. regarding copyright ownership.  The ASF licenses this file
+.. to you under the Apache License, Version 2.0 (the
+.. "License"); you may not use this file except in compliance
+.. with the License.  You may obtain a copy of the License at
+
+..   http://www.apache.org/licenses/LICENSE-2.0
+
+.. Unless required by applicable law or agreed to in writing,
+.. software distributed under the License is distributed on an
+.. "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+.. KIND, either express or implied.  See the License for the
+.. specific language governing permissions and limitations
+.. under the License.
+
+Integrating PyArrow with R
+==========================
+
+Arrow supports exchanging data within the same process through the
+:ref:`c-data-interface`.
```

Comment:
Would it be worth spelling out here that this also works for subprocesses? That might be assumed / known for some with the phrase "within the same process" but I also think others would think / assume that subprocess don't count as the same process for this purpose.

Reply:
Uhm, that's an interesting point. I _think_ that suggesting it here might open a can of worms as what you have access to will depend a lot on _when_ the subprocesses are forked and people might complain that it doesn't work. I would act safe and only explicitly state about same process leaving to the reader knowledge to imagine it can also work for subprocesses under some circumstances.

Reply:
Agreed with @amol- . Users who know and understand what forking are able to make out the consequences by themselves; users who don't don't deserve to be confused :-)